### PR TITLE
Add serialization to settings

### PR DESCRIPTION
**Pull request by sfentress**

### CODE REVIEW

File: src/code/stores/app-settings-store.coffee [hunk 1]
```diff
@@ -3,6 +3,7 @@ AppSettingsActions = Reflux.createActions(
     "showSettingsDialog"
     "close"
     "capNodeValues"
+    "importSettings"
```
Comment (knowuh) [inline src/code/stores/app-settings-store.coffee]: If we are following the actions pattern (don't feel strongly about this) then the method `importSettings` maybe should be renamed `onImportSettings`.

**Comment by knowuh** [comment]
This looks good to me, but see inline comment about `importSettings` action.

File: src/code/stores/app-settings-store.coffee (hunk 1 above)
Comment (sfentress) [inline src/code/stores/app-settings-store.coffee]: Actually it was originally an action, was called as an action, and the method was `onImportSettings`. However, I was tripped up by the async nature of actions during tests.

I actually solved the issue, using Mochas `done` function, which can be asynchronous, but then I felt it was convoluted, and that loading settings should be a synchronous event, like loading nodes and links is.

I switched the importer to call `importSettings` directly, but forgot about the action string above. I will remove it.

**Comment by sfentress** [comment]
Removed the importSettings action string.

**Pull request merged**